stdlib/public/stubs/FoundationHelpers.mm

```diff
@@ -108,8 +117,10 @@ typedef __swift_uint8_t (*getCStringImplPtr)(id,
 
 __swift_uint8_t
 _swift_stdlib_dyld_is_objc_constant_string(const void *addr) {
-  return (SWIFT_RUNTIME_WEAK_CHECK(_dyld_is_objc_constant)
-          && SWIFT_RUNTIME_WEAK_USE(_dyld_is_objc_constant(dyld_objc_string_kind, addr))) ? 1 : 0;
+  return false;
+  //This currently always return false
```

Comment:
There's a few other places where we have overhead from pointlessly supporting these, I'm going to elide them in a followup